.env.example

```
@@ -24,3 +24,7 @@ RESEND_API_KEY=
 RESEND_FROM_EMAIL=onboarding@example.com
 RESEND_FROM_NAME=STDS
 RESEND_BASE_URL=https://api.resend.com
+
+GOOGLE_APPLICATION_CREDENTIALS=somepath
+GCS_BUCKET_NAME=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The GCS_BUCKET_NAME key should go before the GOOGLE_APPLICATION_CREDENTIALS key

(UnorderedKey)